CI.hs

```diff
@@ -101,13 +103,28 @@ genVersionStr = \case
 
 parseOptions :: Opts.Parser Options
 parseOptions = Options
-    <$> Opts.switch
-        ( Opts.long "upload-to-bintray"
-       <> Opts.help "If specified, will try uploading the sdists to Bintray, using credentials in BINTRAY_BASIC_AUTH env var.")
-    <*> Opts.option readFlavor
+    <$> Opts.option readFlavor
         ( Opts.long "ghc-flavor"
        <> Opts.help "The ghc-flavor to test against"
         )
+    <*> Opts.strOption
+        ( Opts.long "da-base-sha"
+       <> Opts.help "DA flavour only. Base commit to use from the GHC repo."
+       <> Opts.showDefault
+       <> Opts.value "ghc-8.8.1-release"
+        )
+    <*> Opts.strOption
```

Comment:
Is there a reason why you don’t allow passing this multiple times one per patch instead of accepting a space-separated option? The latter seems a bit weird but I don’t have strong feelings here so happy to go with your preference.

Reply:
I had not thought of that. I'll have a look at how to do that with optparse.

Reply:
`many` is the function you need for this.

Reply:
Switched to `some`, because I wanted a default value and `many` loops if there is one.